github/ci/prow/files/plugins.yaml

```diff
@@ -112,6 +112,13 @@ plugins:
   - lgtm
   - approve
 
+  kubevirt/cluster-network-addons-operator:
+  - trigger
+  - owners-label
+  - release-note
+  - lgtm
+  - approve
```

Comment:
Could you add `tide`?

Reply:
tide should be enabled by the change in config.yaml above.